validphys2/examples/theory_covariance/working_runcards/covariance_matrix_plots_3fpt.yaml

```diff
@@ -0,0 +1,87 @@
+meta:
+   author: Andrea Barontini
+   keywords: [theory uncertainties, 3f-point]
+   title: NLO 3f-point variations for 5 process types - DIS CC, DIS NC, DY, Top, Jets
+
+default_theory:
+   - theoryid: 163
+
+theoryid: 163
+point_prescription: '3f point'
+
+
+theoryids:
+   from_: scale_variation_theories
+
+dataspecs:
+        - theoryid: 163
+          speclabel: $(\xi_F,\xi_R)=(1,1)$
+        - theoryid: 177
+          speclabel: $(\xi_F,\xi_R)=(2,1)$
+        - theoryid: 176
+          speclabel: $(\xi_F,\xi_R)=(0.5,1)$
+
+normalize_to: 1
+
+fit: 190315_ern_nlo_central_163_global
+use_cuts: "fromfit"
+
+pdf:
+    from_: fit
+
+dataset_inputs:
+  - dataset: NMCPD
+  - dataset: NMC
+  - dataset: SLACP
+  - dataset: SLACD
+  - dataset: BCDMSP
+  - dataset: BCDMSD
+  - dataset: CHORUSNU
+  - dataset: CHORUSNB
+  - dataset: NTVNUDMN
+  - dataset: NTVNBDMN
+  - dataset: HERACOMBNCEM
+  - dataset: HERACOMBNCEP460
+  - dataset: HERACOMBNCEP575
+  - dataset: HERACOMBNCEP820
+  - dataset: HERACOMBNCEP920
+  - dataset: HERACOMBCCEM
+  - dataset: HERACOMBCCEP
+  - dataset: HERAF2CHARM
+  - dataset: CDFZRAP
+  - dataset: D0ZRAP
+  - dataset: D0WEASY
+  - dataset: D0WMASY
+  - dataset: ATLASWZRAP36PB
+  - dataset: ATLASZHIGHMASS49FB
+  - dataset: ATLASLOMASSDY11EXT
+  - dataset: ATLASWZRAP11
+  - dataset: ATLAS1JET11
+  - dataset: ATLASZPT8TEVMDIST
+  - dataset: ATLASZPT8TEVYDIST
+  - dataset: ATLASTTBARTOT
+  - dataset: ATLASTOPDIFF8TEVTRAPNORM
+  - dataset: CMSWEASY840PB
+  - dataset: CMSWMASY47FB
+  - dataset: CMSDY2D11
+  - dataset: CMSWMU8TEV
+  - {dataset: CMSZDIFF12, cfac: [NRM]}
+  - dataset: CMSJETS11
+  - dataset: CMSTTBARTOT
+  - dataset: CMSTOPDIFF8TEVTTRAPNORM
+  - dataset: LHCBZ940PB
+  - dataset: LHCBZEE2FB
+  - {dataset: LHCBWZMU7TEV, cfac: [NRM]}
+  - {dataset: LHCBWZMU8TEV, cfac: [NRM]}
+
+template: template_matrix_plots.md
```

Comment:
For this one should one use the `template_matrix_plots_new` or the old one?

Reply:
The new one ( I fixed now). Anyway all this is just a temporary solution because as you remember we had the problem with the chi2 calculation (now solved I believe). Therefore, the new template is just the same of the old one but without the chi2 functions.